sim/target-agnostic.mk

```diff
@@ -169,38 +186,27 @@ board_dir 	   := $(fpga_dir)/hdk/cl/developer_designs
 fpga_work_dir  := $(board_dir)/cl_$(name_tuple)
 fpga_build_dir := $(fpga_work_dir)/build
 verif_dir      := $(fpga_work_dir)/verif
-fpga_v         := $(fpga_work_dir)/design/cl_firesim_generated.sv
```

Comment:
I don't enumerate individual files anymore because they can cleanly be copied en masse (since they have consistent names).